kubernetes/istio-install/istio-mixer.yaml

```diff
@@ -22,6 +22,50 @@ data:
         params:
       - name: default
         impl: denyChecker
+    attributes:
+      - name: source.name
```

Comment:
I was using the (to my knowledge authoritative) attribute spreadsheet to derive these. `source.name` is separate and distinct from `source.uid` and `source.service` is not defined in that sheet. Can you point me to the attribute producing code in the proxy so I can see what it's producing and how it lines up with that sheet?